src/TorchSharp/NN/Sequential.cs

```diff
@@ -176,6 +181,41 @@ public override nn.Module apply(Action<nn.Module> fn)
                 }
             }
 
+            /// <summary>
+            /// Module indexer.
+            /// </summary>
+            [IndexerName("SequentialItems")]
+            public Sequential this[(int? start, int? end) index] {
+                get {
+                    var result = new Sequential(Array.Empty<torch.nn.Module<Tensor, Tensor>>());
+                    var start = index.start.HasValue ? index.start.Value : 0;
+                    var end = index.end.HasValue ? index.end.Value : _modules.Count;
+
+                    for (var i = start; i < _modules.Count && i < end; i++) {
+                        result.Add(_names[i], _modules[i]);
+                    }
+                    return result;
+                }
+            }
+
+#if !NETSTANDARD2_0_OR_GREATER
+            /// <summary>
+            /// Module indexer.
+            /// </summary>
+            [IndexerName("SequentialItems")]
+            public Sequential this[System.Range index] {
```

Comment:
> public Sequential this[System.Range index] [](http://example.com/codeflow?start=11&length=43)

can we implement this by just calling the other indexer `[start, end]`? just to avoid duplicating the code.

Reply:
Except that the other indexer doesn't have the 'FromEnd' capability that System.Range has.

Reply:
Although I guess the (start,end) tuple would yield the right answer.